tests/goldenfiles/blockfinding.txt

```diff
@@ -5,18 +5,18 @@ data/system/us/seattle/maps/downtown.bin
 data/system/us/seattle/maps/lakeslice.bin
     1006 single blocks (0 failures to blockify), 1 partial merges, 0 failures to blockify partitions
 data/system/us/phoenix/maps/tempe.bin
-    410 single blocks (1 failures to blockify), 0 partial merges, 0 failures to blockify partitions
+    409 single blocks (1 failures to blockify), 0 partial merges, 0 failures to blockify partitions
 data/system/gb/bristol/maps/east.bin
-    817 single blocks (3 failures to blockify), 2 partial merges, 0 failures to blockify partitions
+    820 single blocks (3 failures to blockify), 1 partial merges, 0 failures to blockify partitions
 data/system/gb/leeds/maps/north.bin
-    1930 single blocks (1 failures to blockify), 6 partial merges, 1 failures to blockify partitions
+    1932 single blocks (1 failures to blockify), 6 partial merges, 1 failures to blockify partitions
 data/system/gb/london/maps/camden.bin
-    1034 single blocks (2 failures to blockify), 2 partial merges, 0 failures to blockify partitions
+    1036 single blocks (1 failures to blockify), 2 partial merges, 0 failures to blockify partitions
```

Comment:
I don't really know what to make of these. I guess converting between these representations seems to have slightly altered some topological computation - but not by much?

Reply:
I didn't look into these changes too carefully, except to load the changed maps in the LTN tool and spot-check that adjusting boundaries and default neighbourhood partitioning still looks sane. When I've looked closely into "failures to blockify" in the past, it's always stuff near like 
![Screenshot from 2022-09-12 11-55-44](https://user-images.githubusercontent.com/1664407/189636650-9ef3ce68-dc1a-4e9e-a814-72a2acfc03ef.png)
I'd prefer to spend time fixing the root problems in OSM importing (now part of `osm2streets`, with good unit tests over there) than making things like the block tracing robust to extremely broken inputs